README.md

```diff
@@ -44,66 +44,84 @@
 
    Your site is now running at `http://localhost:8000`!
 
-    *Note: You'll also see a second link: `http://localhost:8000/___graphql`. This is a tool you can use to experiment with querying your data. Learn more about using this tool in the [Gatsby tutorial](https://next.gatsbyjs.org/tutorial/part-five/#introducing-graphiql).*
+   \_\_Note: You'll also see a second link: `http://localhost:8000/___graphql`. This is a tool you can use to experiment with querying your data. Learn more about using this tool in the [Gatsby tutorial](https://next.gatsbyjs.org/tutorial/part-five/#introducing-graphiql).\_\_
```

Comment:
This still doesn't quite seem right: any way we can just leave the wrapper as is (`*`)?

Was the comment about upstream that they use this `\_\_` format (and if so, does it matter if we adopt it here)? Or is there purpose to this formatting that I'm missing otherwise?

Reply:
I'm guessing our remark-lint ruleset was failing when they tried an unscaped `_`, so they escaped it. This should be reverted